style_editor.html

```diff
@@ -7785,6 +7802,26 @@
   blade.addEffect(STATE_ON ? EFFECT_IGNITION : EFFECT_RETRACTION, Math.random() * 0.7 + 0.2);
 }
 
+function OnMoreChange() {
```

Comment:
Doing this on change is weird.
What if you want to trigger the same effect a couple of times?
Seems like it would be better to have a button next to the drop-down for actually triggering the event.

Reply:
Latest update is working, but I had help.